projects/cm_mcu/LocalTasks.c

```diff
@@ -659,58 +660,50 @@ void init_registers_ff()
 
 #ifdef REV2
 
+#define EEPROM_MAX_PER_PAGE 126
+
 static int load_clk_registers(int reg_count, uint16_t reg_page, uint16_t i2c_addrs)
 {
-  uint32_t triplet; // two-byte address and data read from eeprom
-  uint32_t reg0; // a high byte of two-byte address (a page of clock config to keep track when writing data to a clock chip)
-  uint32_t reg1; // a low byte of two-byte address
-  uint32_t data; // data from a config file for each address
-  int HighByte = -1; // keep track when reg0 is changed
-  int status_w = -10; // write data to a clock chip failed or not
-
+  int8_t HighByte = -1; // keep track when reg0 is changed
+  int status_w;
```

Comment:
why do you uninitialize status_w? does it have something to do with deciding between .data or .bass types?